static/src/js/constants/enums.js

```diff
@@ -0,0 +1,21 @@
+export const displayNotificationType = {
+  none: 0,
```

Comment:
We're not considering optimization (as this won't cause any issues in that realm) so if we're introducing enums we'd prefer that the values used are human readable (for debugging) -- let's match the style of `toastType` and `toastPlacement` here and change this to:

```
{
  none: 'none',
  banner: 'banner',
  toast: 'toast'
}
```